### ISSUE TITLE
FuncUnit.com audit

### ISSUE THREAD

**Issue by chasenlehara** [bitovi/funcunit#181]
Go through the website and look for broken demos & links, missing images & videos, spelling & grammar issues, accessibility issues, poor performance, etc.
